sync/evmdriver.go

```diff
@@ -84,18 +87,20 @@ reset:
 	cancellableCtx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	log.Info("Starting sync. syncID: ", syncID, " lastProcessedBlock", lastProcessedBlock)
 	// start downloading
 	downloadCh := make(chan EVMBlock, d.downloadBufferSize)
-	go d.downloader.Download(cancellableCtx, lastProcessedBlock, downloadCh)
+	go d.downloader.Download(cancellableCtx, lastProcessedBlock+1, downloadCh)
 
 	for {
 		select {
 		case b := <-downloadCh:
-			d.log.Debug("handleNewBlock")
+			d.log.Debug("handleNewBlock: ", b.Num, b.Hash, " syncID ", syncID)
```

Comment:
Double check, but should be alright
```suggestion
			d.log.Debug("handleNewBlock:", b.Num, b.Hash, "syncID", syncID)
```

Reply:
Fix: https://github.com/0xPolygon/cdk/pull/56/commits/b754a8b397e29621456a9af15aad0e55984b02cd